apps/fabric-website/src/pages/Controls/MessageBarPage/docs/ios/MessageBarOverview.md

```diff
@@ -0,0 +1,7 @@
+MSNotificationView can be used to present a toast (`.primaryToast` and `.neutralToast` styles) or a notification bar (`.primaryBar`, `.primaryOutlineBar`, and `.neutralBar` styles) with information and actions at the bottom of the screen.
```

Comment:
It's strange that the page is called MessageBar but this control is called MSNotificationView, with no explanation of the link between the two.

Reply:
it was the feedback i got the even thought it is called msnoticationview it should be grouped with web's message bar for consistency